desktop/src/main/java/bisq/desktop/main/dao/economy/dashboard/BsqDashboardView.java

```diff
@@ -122,10 +125,15 @@ public void initialize() {
         createTradeChart();
 
         priceChangeListener = (observable, oldValue, newValue) -> {
-            updatePrice();
-            updateAveragePriceFields(avgPrice90TextField, avgPrice30TextField, false);
-            updateAveragePriceFields(avgUSDPrice90TextField, avgUSDPrice30TextField, true);
-            updateMarketCap();
+            if (System.currentTimeMillis() - lastActivityTime < FASTEST_UPDATE_FREQUENCY_MILLIS) {
+                log.trace("Last activity: {}ms, we ignore this update to prevent freezing the UI.", System.currentTimeMillis() - lastActivityTime);
```

Comment:
What about removing [`updateCounter.set(updateCounter.get() + 1);`](https://github.com/bisq-network/bisq/blob/ad2a329e089ceaaf91e2bfee3e70a41941a3245a/core/src/main/java/bisq/core/provider/price/PriceFeedService.java#L266) from `setBisqMarketPrice` method and calling it once after [these updates](https://github.com/bisq-network/bisq/blob/ad2a329e089ceaaf91e2bfee3e70a41941a3245a/core/src/main/java/bisq/core/provider/price/PriceFeedService.java#L319-L325) instead of this throttling here?